public/css/dashboard.css

```diff
@@ -1,3 +1,6 @@
+body{
+  font-family: "Fira Sans", helvetica;
+}
```

Comment:
Please proceed helvetica with sans-serif in all the occurences. Ex- font-family:"Fira Sans",helvetica,sans-serif;